.gitignore

```diff
@@ -1,7 +1,7 @@
 venv
 __pycache__
 auth_app.db
-app/auth/auth_env.py
-app/registration/reg_env.py
+.env
 db-vars.env
+auth_env.py
```

Comment:
we need this file checked out, it should be just reading from process env, so no secret values there!?